lib/internal/bootstrap/node.js

```diff
@@ -679,13 +679,17 @@ function setupGlobalConsole() {
     value: consoleFromNode,
     writable: true
   });
-  // TODO(joyeecheung): can we skip this if inspector is not active?
+
   if (config.hasInspector) {
-    const inspector =
-      NativeModule.require('internal/console/inspector');
-    inspector.addInspectorApis(consoleFromNode, consoleFromVM);
+    const inspector = NativeModule.require('internal/util/inspector');
     // This will be exposed by `require('inspector').console` later.
     inspector.consoleFromVM = consoleFromVM;
```

Comment:
The console from the VM is only accessible during bootstrap, it gets overridden with our console implementation - an alternative may be to store it in C++ and make it accessible (and cached) through `'internal/util/inspector'`, but otherwise this has to be done in `lib/internal/bootstrap/node.js` one way or another (stored either as a value or through a setter)

Reply:
`consoleFromVM` is passed to `inspector.wrapConsole()` in the line below. Therefore the `consoleFromVM` variable in `lib/internal/util/inspector.js` could be set during the `wrapConsole` function call.

But it's just a nit and is not blocking for me.